eda-env/Lib/site-packages/parso/python/errors.py

```
@@ -0,0 +1,1326 @@
+            message = 'assignment expression cannot be used in a comprehension iterable expression'
+            self.add_issue(namedexpr_test, message=message)
+
+        # defined names
+        exprlist = list()
+
+        def process_comp_for(comp_for):
+            if comp_for.type == 'sync_comp_for':
+                comp = comp_for
+            elif comp_for.type == 'comp_for':
+                comp = comp_for.children[1]
+            exprlist.extend(_get_for_stmt_definition_exprs(comp))
```

Pylint (3.3.7)

[error] Possibly using variable 'comp' before assignment

(E0606)